src/Types/Resources.ts

```diff
@@ -0,0 +1,27 @@
+export interface Resources {
+  label: string; // Name to be displayed to the user
+  name: string; // unique id for reference
+  description: string; // '' for null
+  tags: string[]; // [] for none
+  dos: string[]; // [] for none
+  donts: string[]; // [] for none
+  liveClip: string; // naration video
+  liveAudio: string; // naration audio handle
+  queAudio: string; // Audio to be played as a que
+  eduClip: string[]; // clip id, '' if null
+  images: string[]; // image ids one per count, frist one used live
```

Comment:
There's no need to consolidate these two attributes into one. Let's make it more clear what's going on by splitting this into 2:
```
liveImage: string;
eduImages: string[];
```